File: baekjoon/java/02562. 최댓값.java
import java.util.Scanner;

public class Main {
    private static final int TOTAL = 9;
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        int n, index = 0, max = 0;

        for (int i=0;i<TOTAL;i++){
            n = sc.nextInt();
            if (n>max) {
                max = n;
                index = i;
            }
        }
        System.out.println(max);
        System.out.println(index + 1);

    }
}

File: baekjoon/cpp/02562. 최댓값.cpp
#include <iostream>
#define TOTAL 9
using namespace std;

int main(){
    int n,index, max = 0;

    for (int i=0;i<TOTAL;i++){
        cin >> n;
        if (n>max) {
            max = n;
            index = i;
        }
    }
    cout << max << "\n" << index + 1;
    
    return 0;
}
